crowbar_framework/config/repos.yml

```diff
@@ -0,0 +1,99 @@
+---
+suse-12.0:
+  repos:
+    sles12-pool:
+      name: "SLES12-Pool"
+      required: "mandatory"
+      product_name: "common"
```

Comment:
This is actually important as the `common_available` bit is totally unrelated to the SLES repos. This is for repos that are common to all versions.